horace_core/lattice_functions/refine_crystal.m

```diff
@@ -173,19 +178,18 @@
         fitpar.p=real(fitpar.p);
         distance=real(distance);
     else
-        error('Problem refining crystal orientation: imaginary fit parameters')
+        error('HORACE:lattice_functions:runtime_error', ...
+            'Problem refining crystal orientation: imaginary fit parameters')
     end
 end
 
-rotvec=fitpar.p(7:9);
-rotangle=norm(rotvec)*(180/pi);
-rotmat=rotvec_to_rotmat2(rotvec);
-alatt=fitpar.p(1:3);
-angdeg=fitpar.p(4:6);
+
+rotvec = fitpar.p(7:9);
+alatt  = fitpar.p(1:3);
+angdeg = fitpar.p(4:6);
 distance=sqrt(sum(reshape(distance,3,nv).^2,1))';
```

Comment:
I have no idea what this is trying to do. Should either be commented or more explicit. If it's computing the 3-vector L2 norm then use `norm`

Reply:
ask the author, I am merely reshaping his forumulas into new places. Actually it may be even incorrect L2 norm as hkl are generally non-orthogonal. Or may be I have not notices something.